platform/ext/target/nordic_nrf/nrf5340/gcc/startup_nrf5340_ns.S

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (c) 2009-2016 ARM Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This file is derivative of CMSIS V5.01 startup_ARMCM33.S
+ * Git SHA: 8a1d9d6ee18b143ae5befefa14d89fb5b3f99c75
+ */
+
+    .syntax unified
+    .arch   armv8-m.main
+
+    .section .vectors
+    .align  2
+    .globl  __Vectors
+__Vectors:
+    .long   Image$$ARM_LIB_STACK_MSP$$ZI$$Limit /* Top of Stack */
+
+    .long   Reset_Handler               /* Reset Handler */
+    .long   NMI_Handler                 /* NMI Handler */
+    .long   HardFault_Handler           /* Hard Fault Handler */
+    .long   MemManage_Handler           /* MPU Fault Handler */
+    .long   BusFault_Handler            /* Bus Fault Handler */
+    .long   UsageFault_Handler          /* Usage Fault Handler */
+    .long   SecureFault_Handler         /* Secure Fault Handler */
```

Comment:
@anangl I think this is wrong here. you should not be needing the secure fault handler

Reply:
Right. I'll remove those entries from _ns files.